### ISSUE TITLE
Notification doesn't work with multiple workspaces

### ISSUE THREAD

**Issue by lifeofdan** [solus-project/budgie-desktop#1023]
Pretty much the title. Whenever an app uses the notification action, for example, a messaging client receiving a message, there is a popup. If the chat client is on the workspace you are working in (eg workspace 1) then clicking on the notification popup will switch your active window to the window of the chat program. However, this does not work if your chat client is on a different workspace than you are currently in.

I did try this with multiple chat programs (Discord and Wire) and did multiple reboots to verify it wasn't just me.

So, I guess I am wondering: Is this a bug? Is this intended behaviour? If not intended behaviour is it fixable?

**Comment by lifeofdan** [comment]
Going to leave the issue open as it may be something that gets fixed/added in budgie 11. Would be a good feature. :)

**Comment by orschiro** [comment]
I agree with you that the current behaviour is inconsistent and clicking on the notification icon should either always go to the app or never.
